client/components/ShareButton.js

```diff
@@ -17,16 +19,34 @@ const useStyles = ((canShare) => makeStyles((theme) => ({
 })));
 
 export default () => {
+  const [state, setState] = useState({showSnackbar: false});
   const shareData = {url: window.location.href};
   const classes = useStyles(navigator.canShare && navigator.canShare(shareData))();
-  // TODO: handle navigator.share() failures.
-  return <Button
-      variant='contained'
-      color='primary'
-      startIcon={<ShareIcon />}
-      className={classes.button}
-      onClick={() => navigator.share(shareData)}
-    >
-      Share
-    </Button>
+
+  const share = (shareData) => {
+    navigator.share(shareData).catch((e) => {
+      if (e.name !== "AbortError") {
```

Comment:
nit: single quotes for consistency, same below in the JSX (`variant='filled'`and `severity='error'`)

Reply:
Done.